Add missing loads to config.bzl.

diff --git a/toolchain/config.bzl b/toolchain/config.bzl
index 52e42a2..b0b2894 100644
--- a/toolchain/config.bzl
+++ b/toolchain/config.bzl
@@ -1,5 +1,6 @@
 load("@bazel_tools//tools/build_defs/cc:action_names.bzl", "ACTION_NAMES")
 load("@rules_cc//cc:cc_toolchain_config_lib.bzl", "action_config", "feature", "flag_group", "flag_set")
+load("@rules_cc//cc:defs.bzl", "CcToolchainConfigInfo", "cc_common")

 def _tool_path(bins, toolchain_prefix, tool_name):
     """Generate tool paths for GCC"""
