Do not link kotlin stdlib automatically into all jars.

diff --git a/kotlin/internal/jvm/jvm_deps.bzl b/kotlin/internal/jvm/jvm_deps.bzl
index e6965c9..45d33fe 100644
--- a/kotlin/internal/jvm/jvm_deps.bzl
+++ b/kotlin/internal/jvm/jvm_deps.bzl
@@ -29,7 +29,6 @@ def _jvm_deps(ctx, toolchains, associate_deps, deps = [], deps_java_infos = [],
         associates = associate_deps,
     )
     dep_infos = (
-        [toolchains.kt.jvm_stdlibs] +
         associates.dep_infos +
         deps_java_infos +
         [_java_info(d) for d in deps]
