Do not link kotlin stdlib automatically into all jars.

diff --git a/kotlin/internal/jvm/jvm_deps.bzl b/kotlin/internal/jvm/jvm_deps.bzl
index 746db53..2c9a352 100644
--- a/kotlin/internal/jvm/jvm_deps.bzl
+++ b/kotlin/internal/jvm/jvm_deps.bzl
@@ -31,8 +31,7 @@ def _jvm_deps(ctx, toolchains, associate_deps, deps = [], deps_java_infos = [],
     dep_infos = (
         deps_java_infos +
         [_java_info(d) for d in deps] +
-        associates.dep_infos +
-        [toolchains.kt.jvm_stdlibs]
+        associates.dep_infos
     )

     # Reduced classpath, exclude transitive deps from compilation
