Patches the REPL to terminate inputs with a zero byte.
---
diff --git a/bazel/src/main/java/net/starlark/java/cmd/Main.java b/bazel/src/main/java/net/starlark/java/cmd/Main.java
--- a/bazel/src/main/java/net/starlark/java/cmd/Main.java	(revision 93f38093f8e24875c1d015e67311853756bdb27e)
+++ b/bazel/src/main/java/net/starlark/java/cmd/Main.java	(date 1772718806287)
@@ -65,50 +65,19 @@
   @Nullable
   private static String prompt() {
     StringBuilder input = new StringBuilder();
-    System.out.print(START_PROMPT);
     try {
-      String lineSeparator = "";
-      loop:
       while (true) {
-        String line = reader.readLine();
-        if (line == null) {
-          return null;
+        final var b = reader.read();
+        if (b == '\0') {
+          return input.toString();
+        } else {
+          input.append((char) b);
         }
-        if (line.isEmpty()) {
-          break loop; // a blank line ends the chunk
-        }
-        input.append(lineSeparator).append(line);
-
-        // Read lines until input produces valid statements, unless the last is if/def/for,
-        // which can be multiline, in which case we must wait for a blank line.
-        // TODO(adonovan): parse a compound statement, like the Python and
-        //   go.starlark.net REPLs. This requires a new grammar production, and
-        //   integration with the lexer so that it consumes new
-        //   lines only until the parse is complete.
-        StarlarkFile file = StarlarkFile.parse(ParserInput.fromString(input.toString(), "<stdin>"));
-        if (file.ok()) {
-          List<Statement> stmts = file.getStatements();
-          if (!stmts.isEmpty()) {
-            Statement last = stmts.get(stmts.size() - 1);
-            switch (last.kind()) {
-              case IF:
-              case DEF:
-              case FOR:
-                break; // keep going until blank line
-              default:
-                break loop;
-            }
-          }
-        }
-
-        lineSeparator = "\n";
-        System.out.print(CONTINUATION_PROMPT);
       }
     } catch (IOException e) {
       System.err.format("Error reading line: %s\n", e);
       return null;
     }
-    return input.toString();
   }

   /** Provide a REPL evaluating Starlark code. */
